Tasks/MavenV3/utils.ts

```diff
@@ -0,0 +1,125 @@
+import * as path from 'path';
```

Comment:
There is already mavenutil.ts file which contains some additional utility methods - could it make sense to set up some more precise name for this file to avoid confusion?

Reply:
The few new utils are replaces the old version of ones, but it affects the other old utils which uses it. It requires to refactor the old utils for supporting new ones(f.e., the `writeFile` util) . Since in the future we need to refactor the whole task anyway, make sense to temporarily move the new utils to the separate file